content/blog/how-to-create-a-virtual-network-in-hpe-greenlake-for-private-cloud.md

```diff
@@ -0,0 +1,170 @@
+
+* Navigate to Infrastructure > Networks
+* From the Networks tab, click the ADD drop-down list, select NSX-T Segment
+* From the dialog box, configure the NSX-T segment parameters as follows. For information about additional fields which are not described here, refer [User Guide](https://support.hpe.com/hpesc/public/docDisplay?docId=a00092451en_us&page=GUID-3DCFD624-DFE7-45A8-AFAC-BE004227C7EC.html).
+
+  * **Group**: From the drop-down list, select an infrastructure user group to isolate the network at the group level. The default is Shared (all infrastructure groups)
+  * **Network Service**: Select "NSX-T"
+  * **Name**: Network Name
+  * **ACTIVE**: Select to activate the network. Clear to deactivate the network
+  * **Gateway**: (Optional) Enter the gateway address
+  * **Primary DNS**: (Optional) Enter the primary DNS details
+  * **Secondary DNS**: (Optional) Enter the secondary DNS details
+  * **Connected Gateway**: (Optional) From the drop-down list, select a Tier1 gateway router
+  * **Gateway CIDR**: Enter the Classless Inter-Domain Routing (CIDR) for the logical switch (example: 192.168.0.1/24)
+  * **Transport Zone**: Select Overlay
+  * **Network Pool**: Specify the IP Pool which was created in the prerequisites section
+
+![](/img/fig-3.jpg)
+
+* Click "Save Changes"
+
+On successful creation, the network will list under the "Networks" tab. Use this segment for instance deployment.
+
+![](/img/fig-10.jpg)
+
+## How to create a virtual network with DHCP
+
+Locate HPE GreenLake for private cloud card in the HPE GreenLake Central dashboard and click the Launcher icon to open the HPE GreenLake for private cloud dashboard.
+
+* Navigate to Infrastructure > Networks
+* From the Networks tab, click the ADD drop-down list, select NSX-T Segment
+* From the dialog box, configure the NSX-T segment parameters as follows. For information about additional fields which are not described here, refer [User Guide](https://support.hpe.com/hpesc/public/docDisplay?docId=a00092451en_us&page=GUID-3DCFD624-DFE7-45A8-AFAC-BE004227C7EC.html).
+
+  * **Group**: From the drop-down list, select an infrastructure user group to isolate the network at the group level. The default is Shared (all infrastructure groups)
+  * **Network Service**: Select "NSX-T"
+  * **Name**: Network Name
+  * **ACTIVE**: Select to activate the network. Clear to deactivate the network
+  * **Gateway**: (Optional) Enter the gateway address
+  * **Primary DNS**: (Optional) Enter the primary DNS details
+  * **Secondary DNS**: (Optional) Enter the secondary DNS details
+  * **Connected Gateway**: (Optional) From the drop-down list, select a Tier1 gateway router
+  * **Gateway CIDR**: Enter the Classless Inter-Domain Routing (CIDR) for the logical switch (example: 192.168.0.1/24)
+  * **Transport Zone**: Select Overlay
+* Expand 'Subnet DHCP' Section and update the below fields
+
+  * DHCP Type: Local DHCP Server (default)
+  * DHCP Enabled: Select to Enable
```

Comment:
Because you wrote out "ACTIVE" (as it shows on the screen), I might suggest you do similarly with DHCP ENABLED.